conf/messages

```diff
@@ -505,22 +505,22 @@ numberOfProperties.helpText=All of the property sales or disposals must have the
 numberOfProperties.One=One
 numberOfProperties.MoreThanOne=2 or more
 numberOfProperties.details.summary=What happens on the completion date?
-numberOfProperties.details.p1=<a href="{0}" target="_blank">The completion date (opens in a new window)</a> is usually the day you got the money from the sale or disposal and the new owner got the keys.
-numberOfProperties.error.invalid=Select how many UK properties you are including in this return
-numberOfProperties.error.required=Select how many UK properties you are including in this return
+numberOfProperties.details.p1=<a href="{0}" target="_blank">The completion date (opens in a new window)</a> is usually the day the trust got the money from the sale or disposal and the new owner got the keys.
+numberOfProperties.error.invalid=Select how many properties you are including in this return
+numberOfProperties.error.required=Select how many properties you are including in this return
 
-disposalMethod.title=How did you sell or dispose of the UK property?
+disposalMethod.title=How did the trust sell or dispose of the UK property?
```

Comment:
this will show for individuals also - there needs to be dedicated trust message keys